package.json

```diff
@@ -122,11 +122,15 @@
     "build-benchmarks": "BENCHMARK_VERSION=${BENCHMARK_VERSION:-\"$(git rev-parse --abbrev-ref HEAD) $(git rev-parse --short=7 HEAD)\"} rollup -c bench/rollup_config_benchmarks.js",
     "build-benchmarks-view": "BENCHMARK_VERSION=${BENCHMARK_VERSION:-\"$(git rev-parse --abbrev-ref HEAD) $(git rev-parse --short=7 HEAD)\"} rollup -c bench/rollup_config_benchmarks_view.js",
     "watch-benchmarks": "BENCHMARK_VERSION=${BENCHMARK_VERSION:-\"$(git rev-parse --abbrev-ref HEAD) $(git rev-parse --short=7 HEAD)\"} rollup -c bench/rollup_config_benchmarks.js --watch",
+    "watch-style-benchmarks": "BENCHMARK_VERSION=${BENCHMARK_VERSION:-\"$(git rev-parse --abbrev-ref HEAD) $(git rev-parse --short=7 HEAD)\"} rollup -c bench/rollup_config_style_benchmarks.js --watch",
     "watch-benchmarks-view": "BENCHMARK_VERSION=${BENCHMARK_VERSION:-\"$(git rev-parse --abbrev-ref HEAD) $(git rev-parse --short=7 HEAD)\"} rollup -c bench/rollup_config_benchmarks_view.js --watch",
     "start-server": "st --no-cache -H 0.0.0.0 --port 9966 --index index.html .",
+    "start-bench-server": "st --no-cache -H 0.0.0.0 --port 9966 --index index.versions.html .",
```

Comment:
I've reduced the scripts to just `start-server` and `start-bench` but switching between `bench/versions/` and `bench/styles/` doesn't rebuild the generated files because Rollup isn't run again. Is there a straightforward way to run Rollup when you change URLs? Or is it ok to have to run `start-bench` again?

Reply:
I guess `bench/versions/` and `bench/styles/` should load different bundles, both of which are watched by `start-bench`?